### PR TITLE
fix for mate-applets dependency

### PR DESCRIPTION
**Pull request by palica**
- #64

### CODE REVIEW

File: mate-base/mate-applets/mate-applets-1.26.0.ebuild [hunk 1]
```diff
@@ -39,9 +39,7 @@ COMMON_DEPEND="
 		sys-power/cpupower
 		policykit? ( >=sys-auth/polkit-0.97:0 )
 	)
-	netlink? (
-		dev-libs/libnl:3
-	)
+	dev-libs/libnl:3
```
Comment (oz123) [inline mate-base/mate-applets/mate-applets-1.26.0.ebuild]: is netlink no longer a configuration variable?

Comment (oz123) [reply mate-base/mate-applets/mate-applets-1.26.0.ebuild]: If so this needs to be toggeled together with removing the USE flag at the top.

**Review by oz123 — CHANGES_REQUESTED** [review]

File: mate-base/mate-applets/mate-applets-1.26.0.ebuild (hunk 1 above)
Comment (oz123) [reply mate-base/mate-applets/mate-applets-1.26.0.ebuild]: I see your use flags:
```
* USE:        X abi_x86_64 amd64 cpupower elibc_glibc ipv6 kernel_linux policykit upower userland_GNU
```

Comment (palica) [reply mate-base/mate-applets/mate-applets-1.26.0.ebuild]: OK< I added netlink config option. Please check now.

Comment (oz123) [reply mate-base/mate-applets/mate-applets-1.26.0.ebuild]: Looks. good

**Pull request merged**